admin/tabs/wizard.php

```diff
@@ -119,5 +95,60 @@ public function right_column( $setting ) {
   <!-- End Box -->
         <?php
     }
+    public function load_options( $options, $step ) {
+        ?>
+      <form onsubmit="onClickOptionButton(event)">
+      <table class="widefat striped">
+            <thead>
+              <tr>
+                <th colspan="3"><?php echo 'Set Options' ?></th>
+              </tr>
+            </thead>
+            <tbody>
+            <?php
+            foreach ( $options as $option ){
+                $key = $option->key;
+                $value = $option->value;
+                $db_value = get_option( $key );
+                if ( gettype( $value ) == 'object' ){
+                    $value = json_encode( $value );
+                }
+                if ( gettype( $db_value ) == 'object' ){
+                    $db_value = json_encode( $db_value );
+                }
+                ?>
+              <tr>
+                <td>
+                <?php echo esc_html( $key ); ?>
+                </td>
+                <td>
+                <?php echo esc_html( $value ); ?>
+                </td>
```

Comment:
I don't think we need this column that prints the value to the page. We see the value in the form input, so this is redundant and can be removed.

Reply:
It could be useful to display the current value of that given option though. So if get that option from the db, you can print the value here so it's easy to see what's different and what is going to change.